src/worker/loan/loans/status.js

```diff
@@ -260,6 +260,17 @@ function defineLoanStatusJobs (agenda) {
               console.log('accept or cancel 4')
             }
 
+            if (isArbiter()) {
+              // Warn if loan is about to expire in a day
+              if ( ((Date.now() / 1000) - (lastWarningSent || 0) > 86400) && (currentTime > (parseInt(loanExpiration) - 86400))) {
```

Comment:
As discussed @zpv might be good to have a reminder when 10% of your loan is remaining. Added an issue here: https://github.com/AtomicLoans/agent/issues/85